ci/ci

```diff
@@ -182,29 +329,43 @@ def find_ci():
     return os.path.dirname(cur_dir)
 
 
-def tail_file(build_log_path):
+def tail_file():
```

Comment:
I don't understand this func, why we need to implement tail when we have `docker-compose logs -f <svc name>`?

Reply:
The docker-compose logs will stop at the line where we start systemd. Since the do.sh script is then started as a service. It will not show up in the logs. However we have a build-<svc> log file which we can tail to show the progress.

Reply:
ok got it. please rename it show_build_log or tail_build_log